range-v3/build2file

```diff
@@ -0,0 +1,53 @@
+intf_libs = # Interface dependencies.
+impl_libs = # Implementation dependencies.
+
+./: lib{range-v3}: include/{hxx ixx txx}{**} $impl_libs $intf_libs
+
+# Include the generated version header into the distribution (so that we don't
+# pick up an installed one) and don't remove it when cleaning in src (so that
+# clean results in a state identical to distributed).
+#
+lib{range-v3}: include/range/v3/hxx{version}
+include/range/v3/
+{
+  # @cja struggling to see how to do this in a nice way, but anyway, what is reason for this? seems version.hpp is already set in upstream.
```

Comment:
I think you can remove it. If I remember correctly there was an issue with upstream where the version number in the version header didn't match the  release version number, I reported it and fixed it this way (I'm not sure, but I remember something like this). Anyway if it's consistently provided upstream, let use upstream.